[feature] support OpenBSD

trombik opened the pull request:
this PR supports OpenBSD except 5.9 as the ansible version in 5.9 box is too old.

Commit:
bump all versions to install latest boxes

on Jenkins slaves

Commit:
fix incorrect box name

Commit:
add OpenBSD 5.9

Commit:
bump OpenBSD versions and update meta

Commit:
remove 5.9

Commit:
fix a warning from ansible

Commit:
remove version contsraints

all versions have been installed on all jenkins slaves

mitsururike commented on .kitchen.yml:
```diff
@@ -44,11 +58,38 @@ platforms:
 
+  - name: ubuntu-16.04-amd64
+    driver:
+      box: trombik/ansible-ubuntu-14.04-amd64
```
* typo: s/14/16/
* it should be added to `meta/main.yml` if `xenial` is supported by the PR.

Commit:
bugfix: s/14/16/

update meta
https://github.com/reallyenglish/ansible-role-debug/pull/16#discussion_r146180850
